ci/jjb/jobs/unittests.yaml

```diff
@@ -10,6 +10,7 @@
           values:
             - f24-np
             - f25-np
+            - f26-np
```

Comment:
AFAIK, Pulp master (2.15) doesn't successfully install on Fedora 26. As a result, using it for unit tests might cause lots of failures. You might want to do this addition separately from the clean-ups done elsewhere in this PR.

Reply:
Ahh, I see. It makes sense that the unit tests wouldn't require a working Pulp installation. Thanks for the info. :smile: